### ISSUE TITLE
Flag resource always marks a large number as changed

### ISSUE THREAD

**Issue by bschaeffer** [terraform-providers/terraform-provider-launchdarkly#14]
We have the following defined in terraform:

```
resource "launchdarkly_feature_flag" "some-flag" {
  # ...
  variation_type = "number"
  variations {
    name        = "Some Variation"
    value       = 86400000
  }
}
```

And are seeing the following plan output every single time:

```
~ variations {
  ~ value       = "8.64e+07" -> "86400000"
}
```

If this is an issue with the provider then I think this change should be ignored.

**Comment by bschaeffer** [comment]
FWIW, applying this plan has no effect. `plan->apply->plan` without making any changes reports the same thing over and over again.

**Comment by ldhenry** [comment]
Hey @bschaeffer,

Thanks for bringing this to our attention. It's certainly not the desired behavior. I'll just filed a bug internally and I'll be sure to update this issue when the fix is in.

Thanks,
Henry

**Comment by ldhenry** [comment]
Hey @bschaeffer,

We just released v1.2.0 of the provider which includes a fix for this issue. Feel free to re-open this issue if you are still experiencing problems with large numbers.

Thanks,
Henry
